pinot-controller/src/test/resources/recommenderInput/BloomFilterInput.json

```diff
@@ -12,6 +12,7 @@
         "name": "b",
         "dataType": "DOUBLE",
         "cardinality":6,
+        "singleValueField": false,
```

Comment:
Not sure why this change is needed

Reply:
For multi-value columns, this property has to be available and set to false. If it's not present, it means it's a single value column.